tests/SideBySide.New/DatabaseFixture.cs

```diff
@@ -22,7 +22,6 @@ protected virtual void Dispose(bool disposing)
 			if (disposing)
 			{
 				Connection.Dispose();
-				MySqlConnection.ClearAllPools();
```

Comment:
XUnit instantiates all test classes at the very beginning.  Since pools are shared among tests, `ClearAllPools` wreaks havoc on other tests that are still running.